base/gmp.jl

```diff
@@ -518,6 +534,12 @@ iszero(x::BigInt) = x == Clong(0)
 <(i::Integer, x::BigInt) = cmp(x,i) > 0
 <(x::BigInt, f::CdoubleMax) = isnan(f) ? false : cmp(x,f) < 0
 <(f::CdoubleMax, x::BigInt) = isnan(f) ? false : cmp(x,f) > 0
+isneg(x::BigInt) = x.size < 0
+ispos(x::BigInt) = x.size > 0
```

Comment:
what about `ispositive`  and `isnegative` (ref https://github.com/JuliaLang/julia/pull/20484#issuecomment-278224564) 
might be better more explicitly named